pyqtgraph/Qt/__init__.py

```diff
@@ -415,11 +415,13 @@ def mkQApp(name=None):
     ============== ========================================================
     """
     global QAPP
-    
+    global PALETTE
+
     def onPaletteChange(palette):
         color = palette.base().color().name()
         app = QtWidgets.QApplication.instance()
-        app.setProperty('darkMode', color.lower() != "#ffffff")
+        darkMode = color.lower() < "#e0e0e0"
```

Comment:
Does that work? Doesn't it look at the red channel, mostly?
How about `color.LightnessF() < 0.5` ?

Reply:
Done